File: back/model/src/main/java/com/ftn/sbnz/model/models/Filter.java
package com.ftn.sbnz.model.models;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Filter {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long id;

    @Column(name = "min_price")
    private Integer minPrice;

    @Column(name = "max_price")
    private Integer maxPrice;
    private String team;
    private Integer position;
//    private Integer period;
}

File: front/nba-fantasy/src/app/modules/admin/model/filter.ts
export interface Filter {
    id: number;
    minPrice: number;
    maxPrice: number;
    team: string | null | undefined;
    position: number;
  }
  
  export interface AddFilter {
    minPrice: number;
    maxPrice: number;
    team: string | null | undefined;
    position: number;
  }